pyomo/contrib/parmest/__init__.py

```diff
@@ -12,16 +12,16 @@
 from pyomo.common.deprecation import relocated_module_attribute
```

Comment:
It looks like `black` went a little crazy on this file. Can you please revert these changes?